rotkehlchen/db/schema.py

```diff
@@ -328,6 +328,16 @@
 );
 """
 
+DB_CREATE_EVM_TRANSACTIONS_AUTHORIZATION = """
+CREATE TABLE IF NOT EXISTS evm_transactions_authorization (
+    tx_hash BLOB NOT NULL,
+    chain_id INTEGER NOT NULL,
+    nonce INTEGER NOT NULL,
+    delegated_address TEXT NOT NULL,
+    FOREIGN KEY(tx_hash, chain_id) REFERENCES evm_transactions(tx_hash, chain_id) ON DELETE CASCADE
```

Comment:
shouldn't tx_hash + chain_id be primary keys here?

Reply:
it cannot be just tx_hash + chain id as a transaction can have multiple authorizations. so i included the delegated_address.